chapter_linear-networks/softmax-regression-gluon_vn.md

```diff
@@ -77,9 +77,9 @@ Recall that the softmax function calculates $\hat y_j = \frac{e^{z_j}}{\sum_{i=1
 where $\hat y_j$ is the $j^\mathrm{th}$ element of ``yhat`` and $z_j$ is the $j^\mathrm{th}$ element of the input ``y_linear`` variable, as computed by the softmax.
 -->
 
-Ở ví dụ trước, ta đã tính toán kết quả đầu ra của mô hình và sau đó đã đưa các kết quả này qua hàm mất mát entropy chéo.
+Ở ví dụ trước, ta đã tính toán kết quả đầu ra của mô hình và sau đó đưa các kết quả này qua hàm mất mát entropy chéo.
 Về mặt toán học, cách làm này hoàn toàn có lý.
-Tuy nhiên, từ góc độ tính toán, sử dụng hàm mũ có thể là nguồn gốc của các vấn đề về ổn định số (được bàn trong :numref:`sec_naive_bayes`).
+Tuy nhiên, từ góc độ điện toán, sử dụng hàm mũ có thể là nguồn gốc của các vấn đề về ổn định số học (được bàn trong :numref:`sec_naive_bayes`).
```

Comment:
Hmm. `Điện toán` mình nghe cứ sao sao ấy nhỉ?

Reply:
Điện toán ám chỉ các hình thức tính toán của máy tính. Nếu không ổn, mình có thể sửa lại là `từ góc độ khoa học máy tính`

Reply:
Mình nghĩ đơn giản là `từ góc độ tính toán` là được.